man/systemd-resolve.xml

```diff
@@ -369,6 +369,66 @@
     </variablelist>
   </refsect1>
 
+  <refsect1>
+    <title>Compatibility with <citerefentry><refentrytitle>resolvconf</refentrytitle><manvolnum>8</manvolnum></citerefentry></title>
+
+    <para><command>systemd-resolve</command> is a multi-call binary. When invoked as <literal>resolvconf</literal>
+    (generally achieved by means of a symbolic link of this name to the <command>systemd-resolve</command> binary) it
+    is run in a limited <citerefentry><refentrytitle>resolvconf</refentrytitle><manvolnum>8</manvolnum></citerefentry>
+    compatibility mode. It accepts mostly the same arguments and pushes all data into
+    <citerefentry><refentrytitle>systemd-resolved.service</refentrytitle><manvolnum>8</manvolnum></citerefentry>,
+    similar to how <option>--set-dns=</option> and <option>--set-domain=</option> operate. Note that
+    <command>systemd-resolved.service</command> is the only supported backend, which is different from other
+    implementations of this command. Note that not all operations supported by other implementations are supported
+    natively. Specifically:</para>
+
+    <variablelist>
+      <varlistentry>
+        <term><option>-a</option></term>
+        <term><option>-d</option></term>
+        <term><option>-f</option></term>
+
+        <listitem><para>These switches for registering DNS settings, unregistering them and forcing the operation are
+        fully supported.</para></listitem>
```

Comment:
Frankly, I think there's no choice but to provide a brief summary what these options do. Referring to a "competing" program, especially one that has many semi-compatible implementations, is just too vague for the users. Note that this is only for "supported" options, "ignored" and "unsupported" options below do not have to be explained.